CHANGELOG.md

```diff
@@ -26,6 +26,14 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 # 3.0.3
 
+Note that 3.1.0 will be the first API stable release and interfaces in this release might still change.
+
+* Check out our new [SeqAn3 Cookbook](https://docs.seqan.de/seqan/3.0.3/cookbook.html). It contains a listing of code
+  examples on how to perform particular tasks using the library.
```

Comment:
```suggestion
```

It was in the 3.0.2 changelog because it was all new.
We can either remove it or rewrite it and advertise that it now has many more entries or something.